### DISCUSSION TITLE
Standalone example issue

### DISCUSSION THREAD

**Discussion by JugglingJaxx** [Jakeler/ble-serial#70] [General]
Hi,

I am trying to integrate the ble serial communication into a program I am writing. I tried your standalone example, but just running it as-is I get an error when ble_standalone.py calls the ble.connect() method: Exception has occurred: NameError
name '_cast_uint32_native' is not defined.

How can I resolve this error?

**Comment by Jakeler** [comment]
Can't reproduce this, for me it works as expected. Can you give me more info about your system? Ideally fill out the issue/bug template. Also  please post the full stack trace.

**Comment by JugglingJaxx** [comment]
Thanks for your speedy reply! I figured out it was a dependency issue. It worked outside my virtual environment, so I played with the packages a bit. Uninstalling and reinstalling dbus_fast package resolved the issue.
